Makefile

```diff
@@ -30,25 +30,27 @@ cbmc.zip: cbmc.inc tool-wrapper.inc $(CBMC)/LICENSE $(CBMC)/src/cbmc/cbmc $(CBMC
 	cd $(basename $@) && rm cbmc cbmc-binary goto-cc LICENSE
 	rmdir $(basename $@)
 
-2ls.zip: 2ls.inc tool-wrapper.inc $(2LS)/LICENSE $(2LS)/src/2ls/2ls $(2LS)/src/goto-cc/goto-cc
+2ls.zip: 2ls.inc tool-wrapper.inc $(2LS)/LICENSE $(2LS)/src/2ls/2ls $(2LS)/cbmc/src/goto-cc/goto-cc
 	mkdir -p $(basename $@)
 	$(MAKE) 2ls-wrapper
 	mv 2ls-wrapper $(basename $@)/2ls
 	cp $(2LS)/LICENSE $(basename $@)/
 	cp $(2LS)/src/2ls/2ls $(basename $@)/2ls-binary
-	cp $(2LS)/src/goto-cc/goto-cc $(basename $@)/
+	cp $(2LS)/cbmc/src/goto-cc/goto-cc $(basename $@)/
 	chmod a+rX $(basename $@)/*
 	zip -r $@ $(basename $@)
 	cd $(basename $@) && rm 2ls 2ls-binary goto-cc LICENSE
 	rmdir $(basename $@)
 
-jbmc.zip: jbmc.inc tool-wrapper.inc $(JBMC)/LICENSE $(JBMC)/jbmc/src/jbmc
+jbmc.zip: jbmc.inc tool-wrapper.inc $(JBMC)/LICENSE $(JBMC)/jbmc/src/jbmc/jbmc $(JBMC)/src/goto-cc/goto-cc
 	mkdir -p $(basename $@)
 	$(MAKE) jbmc-wrapper
 	mv jbmc-wrapper $(basename $@)/jbmc
 	cp $(JBMC)/LICENSE $(basename $@)/
 	cp $(JBMC)/jbmc/src/jbmc/jbmc $(basename $@)/jbmc-binary
+	cp $(JBMC)/jbmc/lib/java-models-library/target/core-models.jar $(basename $@)/
+	cp $(JBMC)/src/goto-cc/goto-cc $(basename $@)/
 	chmod a+rX $(basename $@)/*
 	zip -r $@ $(basename $@)
-	cd $(basename $@) && rm jbmc jbmc-binary LICENSE
+	cd $(basename $@) && rm jbmc jbmc-binary goto-cc core-models.jar LICENSE
```

Comment:
I'm surprised goto-cc is of use with jbmc?!

Reply:
You're right. C&P stupidity.

Reply:
Looks better now